.github/workflows/publish.yml

```diff
@@ -2,7 +2,7 @@ name: publish package to npm
 
 on:
   push:
-    branches: [none]
+    branches: [master]
```

Comment:
Just checking on this

Reply:
Yeah this is intended. Turned it off so that my merge from release to here wouldn't trigger a new package publish, now turning it back on.